### PR TITLE
googlePlayAdModal: add install button type experiment

### PR DESCRIPTION
**Pull request by Zolmeister**

### CODE REVIEW

File: src/components/game_player/index.coffee [hunk 1]
```diff
@@ -63,11 +62,10 @@ module.exports = class GamePlayer
         , ENGAGED_PLAY_TIME
 
     User.convertExperiment('game_play').catch log.trace
-
     GooglePlayAdService.shouldShowAdModal().then (shouldShow) ->
       if shouldShow
-        GooglePlayAdService.showAdModal()
-        ga? 'send', 'event', 'game_share_modal', 'show', gameKey
+        GooglePlayAdService.showAdModal().then ->
+          ga? 'send', 'event', 'game_share_modal', 'show', gameKey
```
Comment (austinhallock) [inline src/components/game_player/index.coffee]: this is okay, but it'll break when we remove the exp (and isn't really obvious to the developer that they need to change this line of code when the other method is changed). is it really necessary to put a then?

File: src/stylus/button.styl [hunk 2]
```diff
@@ -81,3 +81,31 @@ $buttonHeight = 40
 
   &:active
     background: $activeBgColor
+
+.button-flat
```
Comment (austinhallock) [inline src/stylus/button.styl]: should code from an experiment be put here? control could win out, which would make this dead code

File: test/unit/services/google_play_ad.coffee [hunk 3]
```diff
@@ -41,16 +41,29 @@ describe 'GooglePlayAdService', ->
 
   describe 'showAdModal', ->
     it 'opens modal', ->
-      GooglePlayAdService.showAdModal()
-      Modal.component.should.exist
-      Modal.closeComponent()
-      GooglePlayAdService.hasAdModalBeenShown = false
+      overrides =
+        User:
+          getExperiments: ->
```
Comment (austinhallock) [inline test/unit/services/google_play_ad.coffee]: I thought we said we'd use Zock here? to override experiments

File: test/unit/services/google_play_ad.coffee [hunk 4]
```diff
@@ -41,16 +41,29 @@ describe 'GooglePlayAdService', ->
...
+          GooglePlayAdService.hasAdModalBeenShown = false
 
     it 'updates hasAdModalBeenShown', ->
-      GooglePlayAdService.hasAdModalBeenShown.should.be.false
-      GooglePlayAdService.showAdModal()
-      GooglePlayAdService.hasAdModalBeenShown.should.be.true
-      GooglePlayAdService.hasAdModalBeenShown = false
+      overrides =
+        User:
+          getExperiments: ->
```
Comment (austinhallock) [inline test/unit/services/google_play_ad.coffee]: ^

File: test/unit/services/google_play_ad.coffee [hunk 5]
```diff
@@ -78,6 +91,8 @@ describe 'GooglePlayAdService', ->
       visitCount = 0
       overrides =
         User:
+          getExperiments: ->
```
Comment (austinhallock) [inline test/unit/services/google_play_ad.coffee]: ^

File: src/components/game_player/index.coffee (hunk 1 above)
Comment (Zolmeister) [inline src/components/game_player/index.coffee]: tests will fail if the method doesn't return a promise

**Comment by Zolmeister** [comment]
updated

**Pull request merged**